README.md

```diff
@@ -221,6 +224,17 @@ Run the integration test suite inside a containerized instance of Firefox. You m
 
 Start a linux VM container with VNC available over `vnc://localhost:5900` with password `secret`. Right click on the desktop and select `Applications > Shell > Bash` and enter `tox -c tests/integration/` to run the integration tests and watch them run in a Firefox instance you can watch and interact with.
 
+## Accessing Remote Settings locally
+
+In development you may wish to approve or reject changes to experiments as if they were on Remote Settings. You can do so here: `http://localhost:8888/v1/admin/`
+
+Default credentials:
+
+- As an admin: `admin` / `admin`
+- As a reviewer: `reviewer` / `review`
+
+These credentials can be found or updated in your `.env` file.
```

Comment:
I don't think this is actually true - or at least I don't see them in [the `.env.sample` file](https://github.com/mozilla/experimenter/blob/main/.env.sample#L28-L29). 

There's a `KINTO_USER` and `KINTO_PASS` setting - but I don't think that *defines* the account, rather it supplies the credentials for the experimenter Django app to use against the local Kinto instance. I was looking around to find a list where the Kinto accounts are defined but have yet to be able to find them

Reply:
Oh wait, after stumbling around anew a bit more, I think I found where the accounts are defined:

https://github.com/mozilla/experimenter/blob/main/app/bin/setup_kinto.py#L7-L10

Reply:
So, actually, *yes* `KINTO_USER` and `KINTO_PASS` from `.env` are used to define one of the accounts - `experimenter` / `experimenter` by default from `.env.sample`. The admin & review accounts are additional hardcoded accounts, though.

Reply:
Might also be interesting to note that these users each [belong to different groups](https://github.com/mozilla/experimenter/blob/main/app/bin/setup_kinto.py#L48-L66) - though [it looks like both groups can write](https://github.com/mozilla/experimenter/blob/main/app/bin/setup_kinto.py#L74-L78), so I'm not sure if that's a distinction with a difference

Reply:
Thanks for looking into this. I'll update to outline the differences as well as where they are hard-coded or in an env var.